src/ec/packages/ec-component-button/button.html.twig

```diff
@@ -25,7 +25,7 @@
 {% set _extra_attributes = '' %}
 {% set _label = label|default('') %}
 {% set _variant = variant|default('primary') %}
-{% set _type = type|default('submit') %}
+{% set _type = type|default('button') %}
```

Comment:
the default should stay as submit, and be overrided when needed.
here it changed it for someother components too (search form, site header).

Reply:
ok.

Reply:
It is solved @emeryro